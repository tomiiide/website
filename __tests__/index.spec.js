import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import IndexPage from "../pages/index";
it("renders correctly", () => {
  const renderer = new ShallowRenderer();
  const result = renderer.render(<IndexPage/>);
  expect(result).toMatchSnapshot();
})
