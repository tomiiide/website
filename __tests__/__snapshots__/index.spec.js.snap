// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<React.Fragment>
  <Head>
    <title>
       Tomide Oladipo 
    </title>
    <link
      href="static/favicon.ico"
      rel="icon"
    />
  </Head>
  <div
    className="homepage"
  >
    <ReactFullpage
      afterRender={[Function]}
      licenseKey="4A80502E-A0AE42C6-A94FDFD9-8E6A4797"
      onLeave={[Function]}
      pluginWrapper={[Function]}
      render={[Function]}
      scrollOverflow={true}
      scrollOverflowReset={true}
      sectionsColor={Array []}
    />
  </div>
</React.Fragment>
`;
